ClrFacade/RDotNetDataConverter.cs

```diff
@@ -32,7 +32,7 @@ private RDotNetDataConverter (string pathToNativeSharedObj)
                 else
                 {
                     dllName = Path.Combine(libDir, Environment.Is64BitProcess ? "x64" : "i386",
```

Comment:
2DO: architecture part of the path sohuld be removed?

Reply:
I think yes, but not 100% sure. let's create an issue for this